### ISSUE TITLE
gravity command to update disk name for docker device

### ISSUE THREAD

**Issue by UnderGreen** [gravitational/gravity#1585]
Gravity should provide a command to rename disk name for docker devices(devicemapper, pre-5.2) in the working cluster. 

As an example, NVME disks in AWS could be renamed after node restart and if a restart happens during the upgrade it could lead to the issue with the plan because device in a plan does not exist anymore.

Currently, the problem is solvable by manually patching ETCD state, which is not convenient and could be error-prone.

**Comment by r0mant** [comment]
A bit more information about how nvme disks change b/w restarts:

```
[centos@ip-172-31-41-223 ~]$ lsblk
NAME        MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
nvme0n1     259:1    0   50G  0 disk
└─nvme0n1p1 259:2    0   50G  0 part /
nvme1n1     259:0    0  100G  0 disk
nvme2n1     259:3    0   25G  0 disk

// reboot

[centos@ip-172-31-41-223 ~]$ lsblk
NAME        MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
nvme0n1     259:2    0   50G  0 disk
└─nvme0n1p1 259:3    0   50G  0 part /
nvme1n1     259:0    0   25G  0 disk
nvme2n1     259:1    0  100G  0 disk
```

This also leads to older Gravity (that uses devicemapper) not being able to install on such nodes because we can't rely on the device name (/dev/nvme*n1) on being stable.

**Comment by r0mant** [comment]
Released in [5.5.48](https://gravitational.com/gravity/docs/changelog/#5548-lts-june-15th-2020).
